TYPO3.Media/Classes/TYPO3/Media/Command/MediaCommandController.php

```diff
@@ -123,6 +175,31 @@ public function clearThumbnailsCommand()
     }
 
     /**
+     * Generate uninitialized asynchronous thumbnails
+     *
+     * @param integer $limit
+     * @return void
+     */
+    public function generateThumbnailsCommand($limit = null)
```

Comment:
yeah it's been bugging me as well, but haven't thought of something better.. could be ``renderThumbnails`` instead maybe